docker-compose.yml

```diff
@@ -58,30 +62,41 @@ services:
       internal:
 
   # Underpass API
-  underpass-api:
+  api:
+    image: "ghcr.io/hotosm/underpass/api:${APP_VERSION:-debug}"
     container_name: "underpass_api"
     build:
      context: .
      dockerfile: docker/underpass-api.dockerfile
-    # volumes:
-    #   - ./python:/code/api
+     target: debug
+     args:
+        APP_VERSION: ${APP_VERSION:-debug}
+    depends_on: [underpass]
+    volumes:
+      - ./python:/code
     ports:
-      - "8000:8000"
+      - "${API_PORT:-8000}:8000"
     networks:
       internal:
     environment:
       - UNDERPASS_API_DB=postgresql://underpass:underpass@postgis/underpass
 
   # Underpass UI
-  underpass-ui:
+  ui:
+    image: "ghcr.io/hotosm/underpass/ui:${APP_VERSION:-debug}"
     container_name: "underpass_ui"
     build:
      context: .
      dockerfile: docker/underpass-ui.dockerfile
+     target: debug
+     args:
+        APP_VERSION: ${APP_VERSION:-debug}
+    # # Mount underpass-ui repo
     # volumes:
-    #   - js:/code
+    #   - ../underpass-ui/src:/code/src
+    #   - ../underpass-ui/playground:/code/playground
     ports:
-      - "5000:5000"
+      - "${UI_PORT:-5000}:8080"
```

Comment:
Hey @spwoodcock , I've tested this PR and the UI is working only when I change this to  ```- "${UI_PORT:-5000}:5000"```